config/locales/es/activemodel.yml

```diff
@@ -4,7 +4,7 @@ es:
       verification:
         residence: "Residencia"
         sms: "SMS"
-      local_census_records/import:
+      local_census_record/import:
```

Comment:
Actually, I think this key is OK, and it's the `config/locales/en/activemodel.yml` file which needs to be modified, according to the console on the master branch:

```
> I18n.locale = :en
=> :en
> LocalCensusRecords::Import.model_name.human
=> "Import"
> I18n.locale = :es
=> :es
> LocalCensusRecords::Import.model_name.human
=> "Importación de registros del censo local"
```